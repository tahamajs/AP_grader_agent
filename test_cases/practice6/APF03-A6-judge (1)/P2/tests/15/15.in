POST reserve ? restaurant_name "lidoma" table_id "1" start_time "15" end_time "17" foods "joojeh"
POST signup ? username "amir" password "rfz82"
POST reserve ? restaurant_name "ladoma" table_id "1" start_time "15" end_time "17" foods "joojeh"
POST increase_budget ? amount 1000
POST reserve ? restaurant_name "lidoma" table_id "1" start_time "15" end_time "17" foods "joojeh"