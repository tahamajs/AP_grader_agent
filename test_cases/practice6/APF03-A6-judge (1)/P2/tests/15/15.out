Permission Denied
OK
Not Found
OK
Reserve ID: 1
Table 1 for 15 to 17 in lidoma
Original Price: 320
Order Amount Discount: 0
Total ItemSpecific Discount: 0
First Order Discount: 40
Total Discount: 40
Total Price: 280